src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.box {
+  width: $squeare;
+  padding-top: 30px;
+  margin: 0 auto;
+}
```

Comment:
Better do the same with mix. make the `body `a block and add `body__calendar` to your calendar